### ISSUE TITLE
Push of directory changes file modification times

### ISSUE THREAD

**Issue by neilromig** [rakyll/drive#50]
I am pushing a directory to my Google Drive account and files in uploaded directories and subdirectories all have their times (modification, access and change) set to the time drive has uploaded them. Those not uploaded remain unaltered.

**Comment by codecentaur** [comment]
I agree also.  It would be nice if drive did not change modification times.  If it is necessary to keep the modification times in sync then it should send the modification time along with the file.

By the way, thank you @odeke-em for keeping this project alive!

**Comment by odeke-em** [comment]
Please checkout the latest from my fork since the project moved there, and then reopen the issue there if it persists. @mattlbaker glad you are able to use drive.
